app-component.html

```diff
@@ -9,10 +9,12 @@
   </div>
 
   <div ng-show="!$ctrl.route.changing" class="br-content">
-    <div ng-if="$ctrl.config.userAgent.obsolete" class="alert alert-danger">
-      Your browser ({{$ctrl.config.userAgent.family}}
-      {{$ctrl.config.userAgent.major}}) is <strong>out of date</strong>.
-      Please <a href="http://www.updateyourbrowser.net/">update your browser.</a>
+    <div ng-if="$ctrl.config.userAgent.obsolete" class="container">
```

Comment:
I'm wondering if this `container` class should be `$root.app.ngClass.rootContainer || 'container'`, like on the new L20?